data/migrations/20181011153620_unit_of_measurement_table.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('UofM', function(tbl) {
+        tbl.increments();
+        tbl.string('Unit', 64).notNullable();
+        tbl.integer('quantity_id').unsigned().references('id').inTable('quantity');
+    })
+};
```

Comment:
You can also put the unit of measure on the `ingredients_in_recipes` table once you have created it.